docs/source/core.rst

```diff
@@ -49,6 +49,16 @@ Stores configurations and makes them available via `useConfig`_ hook.
 =====================
 
 
+<LocalMulticallProvider>
+=====================
+
+Ensures that a Multicall contract address is available when developing on a local chain. 
+A Multicall contract will be deployed when an address is not defined in the `Config`_.
```

Comment:
A Multicall contract will be deployed when a multicall address on a local chainID is not defined in the `Config`_.